src/projects.ts

```diff
@@ -158,4 +171,28 @@ export async function createProject(client: DatanestClient, projectData: Project
         project: Project;
         project_link: string;
     };
-}
\ No newline at end of file
+}
+
+export async function patchProject(client: DatanestClient, projectData: Partial<ProjectCreationData>) {
+    const response = await client.patch('v1/projects', projectData);
+    if (response.status !== 200) {
+        throw new DatanestResponseError(`Failed to create project: ${response.status}`, response.status, await response.json());
+    }
+
+    const data = await response.json();
+    return data as {
+        project: Project;
+        project_link: string;
+    };
+}
+
+/**
+ * Archive a project to hide it from users, it will be automatically deleted after some time.
+ * @param client 
+ * @param projectUuid 
+ * @returns 
+ */
+export async function archiveProject(client: DatanestClient, projectUuid: string) {
```

Comment:
Why is delete being equated to archive ?

Reply:
nvm, understood this now.

Reply:
Allows us to restore if projects are accidentally deleted. We do not currently allow users to delete projects, they are instead deleted after being archived for 6 months, by default.